CMakeLists.txt

```diff
@@ -270,6 +276,10 @@ if(VELOX_ENABLE_PARQUET)
   set(VELOX_ENABLE_ARROW ON)
 endif()
 
+if(VELOX_ENABLE_COMPRESSION_LZ4)
+  add_definitions(-DVELOX_ENABLE_COMPRESSION_LZ4)
+endif()
```

Comment:
Move this into common/compression and use `velox_compile_definitions(velox_common_compression PUBLIC VELOX_ENABLE_COMPRESSION_LZ4)` (-D is stripped and readded anyway)